zerver/views/presence.py

```diff
@@ -57,6 +57,9 @@ def update_active_status_backend(request, user_profile, status=REQ(),
                                  ping_only=REQ(validator=check_bool, default=False),
                                  new_user_input=REQ(validator=check_bool, default=False)):
     # type: (HttpRequest, UserProfile, str, bool, bool) -> HttpResponse
+    if user_profile.is_bot:
+        return json_error(_('Presence cannot be updated for bot users.'))
```

Comment:
Should error messages for similar things generally be unique or non-unique? E.g. both `get_presence_backend` and `update_active_status_backend` have a check for `user_profile.is_bot`.